blueocean-dashboard/src/main/js/creation/bitbucket/server/steps/BbChooseServerStep.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { PropTypes } from 'react';
+import { observer } from 'mobx-react';
+import { Dropdown, FormElement } from '@jenkins-cd/design-language';
+
+import FlowStep from '../../../flow2/FlowStep';
+import BbAddServerDialog from '../BbAddServerDialog';
+
+
+let t = null;
+
+
+@observer
+class BbChooseServerStep extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedServer: null,
+            showAddServerDialog: false,
+            urlErrorMsg: null,
+        };
+
+        this.dropdown = null;
+    }
+
+    componentWillMount() {
+        t = this.props.flowManager.translate;
+    }
+
+    _onChangeServerDropdown(option) {
+        const { serverManager } = this.props.flowManager;
+        serverManager.validateVersion(option.id)
```

Comment:
Is there some way we could validate the server version _before_ we actually create it? It seems non-ideal to let a user create a server that can't actually be used. Is this handling a scenario where off in the future the minimum version of Bitbucket Server gets raised and we need prevent users from selecting servers that were added a long time ago?